config/settings.local.yml.example

```diff
@@ -15,3 +15,4 @@
 # NOTE: This file is excluded by railsconfig in the test env.
 # Use config/settings/test.local.yml instead.
 # See "Note" at https://github.com/rubyconfig/config#developer-specific-config-files
+
```

Comment:
I'd delete this line in the k8s branch.

Reply:
PR up for k8s branch - https://github.com/department-of-veterans-affairs/vets-api/pull/15788/files